cmake_format/vscode_extension/src/extension.ts

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
+import * as child_process from 'child_process';
```

Comment:
This change looks arbitrary, what is the reason for it?

Reply:
[same reasoning as here](https://github.com/cheshirekow/cmake_format/pull/187#discussion_r405745539)